webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2920,14 +2925,29 @@ var userOverride = {
                         'id',
                         'km-' + preLeadCollection.field.toLowerCase()
                     );
-                    kmChatInput.setAttribute(
-                        'type',
-                        preLeadCollection.type || 'text'
-                    );
-                    kmChatInput.setAttribute(
-                        'name',
-                        'km-' + preLeadCollection.field.toLowerCase()
-                    );
+                    if (
+                        preLeadCollection.hasOwnProperty('type') &&
+                        preLeadCollection.type.toLowerCase() === 'number' &&
+                        ['mobile', 'phone'].indexOf(
+                            preLeadCollection.field.toLowerCase()
+                        ) > -1
+                    ) {
+                        kmChatInput.setAttribute('id', 'km-phone');
```

Comment:
move this outside if else
kmChatInput.setAttribute(
                            'type',
                            preLeadCollection.type || 'text'
                        );
and remove this type set attribute from both places